packages/itwin/measure-tools/src/api/Polygon.ts

```diff
@@ -66,10 +68,19 @@ export class Polygon {
     this._textMarker.applyStyle(this._styleSet.getTextStyle(WellKnownTextStyleType.AreaMeasurement));
   }
 
-  constructor(points: Point3d[], copyPoints: boolean = true, styleSet?: StyleSet) {
+  public set sheetToWorldScale(scale: number | undefined) {
+    this._sheetToWorldScale = scale ? (scale * scale): undefined;
```

Comment:
Make sure to check against `undefined` when dealing with numbers, eg. `undefined !== scale` ? (scale * scale) : undefined

Reply:
Also, keep it as the raw value, do not square it. That makes it very inconsistent and unpredictable.

ex:
```ts
const p: Polygon
p.sheetToWorldScale = 3;
console.log(p.sheetToWorldScale); // 9
p.sheetToWorldScale = p.sheetToWorldScale;
console.log(p.sheetToWorldScale); // 81
```
I think you get the point.

Reply:
Good catch